schema.sql

```diff
@@ -6,7 +6,23 @@
     escape_attempts INTEGER,
     neutered BOOLEAN,
     weight_kg DECIMAL(10,2));
+  
+ALTER TABLE animals
+ADD COLUMN species VARCHAR(255);
+
+CREATE TABLE owners (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    full_name VARCHAR(255),
+    age INT
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+ALTER TABLE animals
+ADD COLUMN species_id INT REFERENCES species(id);
 
-    
 ALTER TABLE animals
-ADD COLUMN species VARCHAR(255);
\ No newline at end of file
+ADD COLUMN owner_id INT REFERENCES owners(id);
```

Comment:
Please note that you are supposed to Remove column species and Add column species_id, you already did the latter but instead of add column here you should have drop. 😉